workshops/ReinforcementLearning2021/jupyter/ice_env.py

```diff
@@ -0,0 +1,109 @@
+import random
+import torch
+
+SLIP_CHANCE = 0.05
+
+# Gridworld. ' ' is an ice cell, '*' a crack, 'T' the treasure, and 'G' the goal.
+GRID = [
+    [' ', ' ', ' ', 'G'],
+    [' ', '*', ' ', '*'],
+    [' ', ' ', 'T', '*'],
+    [' ', '*', '*', '*'],
+]
+
+ACTIONS = ['^', 'v', '<', '>']
+REWARD = {' ': -5.0, '*': -10.0, 'T': 20.0, 'G': 100.0}
```

Comment:
nit: why have this when you'll initialize this stuff in the constructor?

Reply:
It's a default configuration for the assignment they will be completing. Left purely for ease of reference during the workshop as time is a bit tight, can remove if necessary.